app/models/template.rb

```diff
@@ -31,8 +31,9 @@ def self.template_includes
     []
   end
 
-  def preview_host_collection
-    Host.authorized(:view_hosts).order(:name).limit(100)
```

Comment:
I think it might make sense to keep the default limit in case a plugin using this function doesn't override it so that not all hosts will be loaded, what do you say?